File: invesdwin-scripting-parent/invesdwin-scripting-julia/invesdwin-scripting-julia-runtime-contract/src/test/java/de/invesdwin/context/julia/runtime/contract/InputsAndResultsTestShort.jl
println("getShort")
if isdefined(Main, :getShort) && !isnothing(getShort)
	error("getShort already defined!")
end
getShort = putShort
println(typeof(getShort))
println(getShort)
if typeof(getShort) != Int16
	error("getShort not Int16!")
end

println("getShortVector")
if isdefined(Main, :getShortVector) && !isnothing(getShortVector)
	error("getShortVector already defined!")
end
getShortVector = putShortVector
println(eltype(getShortVector))
println(getShortVector)
if eltype(getShortVector) != Int16
	error("getShortVector not Int16!")
end

println("getShortVectorAsList")
if isdefined(Main, :getShortVectorAsList) && !isnothing(getShortVectorAsList)
	error("getShortVectorAsList already defined!")
end
getShortVectorAsList = putShortVectorAsList
println(eltype(getShortVectorAsList))
println(getShortVectorAsList)
if eltype(getShortVectorAsList) != Int16
	error("getShortVectorAsList not Int16!")
end

println("getShortMatrix")
if isdefined(Main, :getShortMatrix) && !isnothing(getShortMatrix)
	error("getShortMatrix already defined!")
end
getShortMatrix = putShortMatrix
println(eltype(getShortMatrix))
println(getShortMatrix)
if eltype(getShortMatrix) != Int16
	error("getShortMatrix not Int16!")
end

println("getShortMatrixAsList")
if isdefined(Main, :getShortMatrixAsList) && !isnothing(getShortMatrixAsList)
	error("getShortMatrixAsList already defined!")
end
getShortMatrixAsList = putShortMatrixAsList
println(eltype(getShortMatrixAsList))
println(getShortMatrixAsList)
if eltype(getShortMatrixAsList) != Int16
	error("getShortMatrixAsList not Int16!")
end

File: invesdwin-scripting-parent/invesdwin-scripting-runtime-lua/src/test/java/de/invesdwin/scripting/runtime/lua/tests/InputsAndResultsTestShort.lua
arraysToString = java.method(java.import('de.invesdwin.util.collections.Arrays'),'toString','java.lang.Object')

print("getShort")
if(getShort ~= nil) then
	error("getShort already defined!")
end
getShort = putShort
getShortType = type(getShort)
print(getShortType)
print(getShort)
if(getShortType ~= "number") then
	error("getShort not Short!")
end

print("getShortVector")
if(getShortVector ~= nil) then
	error("getShortVector already defined!")
end
getShortVector = putShortVector
getShortVectorType = type(getShortVector[1])
print(getShortVectorType)
print(arraysToString(getShortVector))
if(getShortVectorType ~= "number") then
	error("getShortVector not Short!")
end

print("getShortVectorAsList")
if(getShortVectorAsList ~= nil) then
	error("getShortVectorAsList already defined!")
end
getShortVectorAsList = putShortVectorAsList
getShortVectorAsListType = type(getShortVectorAsList[1])
print(getShortVectorAsListType)
print(arraysToString(getShortVectorAsList))
if(getShortVectorAsListType ~= "number") then
	error("getShortVectorAsList not Short!")
end

print("getShortMatrix")
if(getShortMatrix ~= nil) then
	error("getShortMatrix already defined!")
end
getShortMatrix = putShortMatrix
getShortMatrixType = type(getShortMatrix[1][1])
print(getShortMatrixType)
print(arraysToString(getShortMatrix))
if(getShortMatrixType ~= "number") then
	error("getShortMatrix not Short!")
end

print("getShortMatrixAsList")
if(getShortMatrixAsList ~= nil) then
	error("getShortMatrixAsList already defined!")
end
getShortMatrixAsList = putShortMatrixAsList
getShortMatrixAsListType = type(getShortMatrixAsList[1][1])
print(getShortMatrixAsListType)
print(arraysToString(getShortMatrixAsList))
if(getShortMatrixAsListType ~= "number") then
	error("getShortMatrixAsList not Short!")
end
